### PR TITLE
Added skiller_skg50_s2 keyboard

### PR DESCRIPTION
**Pull request by itarze**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sharkoon/skiller_sgk50_s2/info.json [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+                {"matrix": [4, 4], "x": 4.25, "y": 4},
+                {"matrix": [4, 5], "x": 5.25, "y": 4},
+                {"matrix": [4, 6], "x": 6.25, "y": 4},
+                {"matrix": [4, 7], "x": 7.25, "y": 4},
+                {"matrix": [4, 8], "x": 8.25, "y": 4},
+                {"matrix": [4, 9], "x": 9.25, "y": 4},
+                {"matrix": [4, 10], "x": 10.25, "y": 4},
+                {"matrix": [4, 11], "x": 11.25, "y": 4},
+                {"matrix": [4, 12], "x": 12.25, "y": 4, "w": 1.75},
+                {"matrix": [4, 14], "x": 14.25, "y": 4},
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s2/info.json:286]:
```suggestion
                {"matrix": [4, 14], "x": 14, "y": 4},
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Comment by dunk2k** [comment]
Does this PCB have ISO (form factor) support?

File: keyboards/sharkoon/skiller_sgk50_s2/info.json [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x3662",
+        "suspend_wakeup_delay": 1000,
+        "vid": "0x6332"
+    },
+    "ws2812": {
+        "pin": "A8"
+    },
+    "layouts": {
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s2/info.json:199]:
```suggestion
    "community_layouts": ["96_ansi"],
    "layouts": {
```

File: keyboards/sharkoon/skiller_sgk50_s2/info.json [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+                {"matrix": [5, 10], "x": 10, "y": 5},
+                {"matrix": [5, 11], "x": 11, "y": 5},
+                {"matrix": [5, 12], "x": 12, "y": 5},
+                {"matrix": [5, 13], "x": 13, "y": 5},
+                {"matrix": [5, 14], "x": 14, "y": 5},
+                {"matrix": [5, 15], "x": 15, "y": 5},
+                {"matrix": [5, 16], "x": 16, "y": 5},
+                {"matrix": [5, 17], "x": 17, "y": 5}
+            ]
+        }
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s2/info.json:304]:
```suggestion
        },
        "LAYOUT_96_ansi": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0},
                {"matrix": [0, 14], "x": 14, "y": 0},
                {"matrix": [0, 15], "x": 15, "y": 0},
                {"matrix": [0, 16], "x": 16, "y": 0},
                {"matrix": [0, 17], "x": 17, "y": 0},
                {"matrix": [0, 18], "x": 18, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1},
                {"matrix": [1, 1], "x": 1, "y": 1},
                {"matrix": [1, 2], "x": 2, "y": 1},
                {"matrix": [1, 3], "x": 3, "y": 1},
                {"matrix": [1, 4], "x": 4, "y": 1},
                {"matrix": [1, 5], "x": 5, "y": 1},
                {"matrix": [1, 6], "x": 6, "y": 1},
                {"matrix": [1, 7], "x": 7, "y": 1},
                {"matrix": [1, 8], "x": 8, "y": 1},
                {"matrix": [1, 9], "x": 9, "y": 1},
                {"matrix": [1, 10], "x": 10, "y": 1},
                {"matrix": [1, 11], "x": 11, "y": 1},
                {"matrix": [1, 12], "x": 12, "y": 1},
                {"matrix": [1, 13], "x": 13, "y": 1, "w": 2},
                {"matrix": [1, 15], "x": 15, "y": 1},
                {"matrix": [1, 16], "x": 16, "y": 1},
                {"matrix": [1, 17], "x": 17, "y": 1},
                {"matrix": [1, 18], "x": 18, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.5},
                {"matrix": [2, 1], "x": 1.5, "y": 2},
                {"matrix": [2, 2], "x": 2.5, "y": 2},
                {"matrix": [2, 3], "x": 3.5, "y": 2},
                {"matrix": [2, 4], "x": 4.5, "y": 2},
                {"matrix": [2, 5], "x": 5.5, "y": 2},
                {"matrix": [2, 6], "x": 6.5, "y": 2},
                {"matrix": [2, 7], "x": 7.5, "y": 2},
                {"matrix": [2, 8], "x": 8.5, "y": 2},
                {"matrix": [2, 9], "x": 9.5, "y": 2},
                {"matrix": [2, 10], "x": 10.5, "y": 2},
                {"matrix": [2, 11], "x": 11.5, "y": 2},
                {"matrix": [2, 12], "x": 12.5, "y": 2},
                {"matrix": [2, 13], "x": 13.5, "y": 2, "w": 1.5},
                {"matrix": [2, 15], "x": 15, "y": 2},
                {"matrix": [2, 16], "x": 16, "y": 2},
                {"matrix": [2, 17], "x": 17, "y": 2},
                {"matrix": [2, 18], "x": 18, "y": 2, "h": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 1.75},
                {"matrix": [3, 2], "x": 1.75, "y": 3},
                {"matrix": [3, 3], "x": 2.75, "y": 3},
                {"matrix": [3, 4], "x": 3.75, "y": 3},
                {"matrix": [3, 5], "x": 4.75, "y": 3},
                {"matrix": [3, 6], "x": 5.75, "y": 3},
                {"matrix": [3, 7], "x": 6.75, "y": 3},
                {"matrix": [3, 8], "x": 7.75, "y": 3},
                {"matrix": [3, 9], "x": 8.75, "y": 3},
                {"matrix": [3, 10], "x": 9.75, "y": 3},
                {"matrix": [3, 11], "x": 10.75, "y": 3},
                {"matrix": [3, 12], "x": 11.75, "y": 3},
                {"matrix": [3, 13], "x": 12.75, "y": 3, "w": 2.25},
                {"matrix": [3, 15], "x": 15, "y": 3},
                {"matrix": [3, 16], "x": 16, "y": 3},
                {"matrix": [3, 17], "x": 17, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 2.25},
                {"matrix": [4, 2], "x": 2.25, "y": 4},
                {"matrix": [4, 3], "x": 3.25, "y": 4},
                {"matrix": [4, 4], "x": 4.25, "y": 4},
                {"matrix": [4, 5], "x": 5.25, "y": 4},
                {"matrix": [4, 6], "x": 6.25, "y": 4},
                {"matrix": [4, 7], "x": 7.25, "y": 4},
                {"matrix": [4, 8], "x": 8.25, "y": 4},
                {"matrix": [4, 9], "x": 9.25, "y": 4},
                {"matrix": [4, 10], "x": 10.25, "y": 4},
                {"matrix": [4, 11], "x": 11.25, "y": 4},
                {"matrix": [4, 12], "x": 12.25, "y": 4, "w": 1.75},
                {"matrix": [4, 14], "x": 14, "y": 4},
                {"matrix": [4, 15], "x": 15, "y": 4},
                {"matrix": [4, 16], "x": 16, "y": 4},
                {"matrix": [4, 17], "x": 17, "y": 4},
                {"matrix": [4, 18], "x": 18, "y": 4, "h": 2},
                {"matrix": [5, 0], "x": 0, "y": 5, "w": 1.25},
                {"matrix": [5, 1], "x": 1.25, "y": 5, "w": 1.25},
                {"matrix": [5, 2], "x": 2.5, "y": 5, "w": 1.25},
                {"matrix": [5, 6], "x": 3.75, "y": 5, "w": 6.25},
                {"matrix": [5, 10], "x": 10, "y": 5},
                {"matrix": [5, 11], "x": 11, "y": 5},
                {"matrix": [5, 12], "x": 12, "y": 5},
                {"matrix": [5, 13], "x": 13, "y": 5},
                {"matrix": [5, 14], "x": 14, "y": 5},
                {"matrix": [5, 15], "x": 15, "y": 5},
                {"matrix": [5, 16], "x": 16, "y": 5},
                {"matrix": [5, 17], "x": 17, "y": 5}
            ]
        }
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
add Community Layout support

**Comment by itarze** [comment]
> Does this PCB have ISO (form factor) support?
> 
> in particular - ISO Hash and ISO Enter keys

yes.

**Comment by dunk2k** [comment]
> > Does this PCB have ISO (form factor) support?
> > in particular - ISO Hash and ISO Enter keys
> 
> yes.

Great stuff.
Would I be correct that the matrix co-ordinates for the following keys are identical?:
- ANSI Backslash <> ISO Hash
- ANSI Enter <> ISO Enter

File: keyboards/sharkoon/skiller_sgk50_s2/info.json [hunk 4]
```diff
@@ -0,0 +1,411 @@
...
+        "pid": "0x3662",
+        "suspend_wakeup_delay": 1000,
+        "vid": "0x6332"
+    },
+    "ws2812": {
+        "pin": "A8"
+    },
+    "community_layouts": ["96_ansi"],
+    "layouts": {
+        "LAYOUT": {
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s2/info.json:201]:
```suggestion
        "LAYOUT_all": {
```

File: keyboards/sharkoon/skiller_sgk50_s2/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    [0] = LAYOUT(
+        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PAUS, KC_HOME, KC_END,    KC_PGUP,  KC_PGDN,  KC_DEL,
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_NUM,    KC_PSLS,  KC_PAST,  KC_PMNS,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_P7,     KC_P8,    KC_P9,    KC_PPLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           KC_P4,     KC_P5,    KC_P6,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT, KC_UP,   KC_P1,     KC_P2,    KC_P3,    KC_PENT,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT,   KC_P0,    KC_PDOT
+    ),
+
+    [1] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s2/keymaps/default/keymap.c:17]:
```suggestion
    [0] = LAYOUT_all(
        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PAUS, KC_HOME, KC_END,    KC_PGUP,  KC_PGDN,  KC_DEL,
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_NUM,    KC_PSLS,  KC_PAST,  KC_PMNS,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_P7,     KC_P8,    KC_P9,    KC_PPLS,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           KC_P4,     KC_P5,    KC_P6,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT, KC_UP,   KC_P1,     KC_P2,    KC_P3,    KC_PENT,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT,   KC_P0,    KC_PDOT
    ),

    [1] = LAYOUT_all(
```

File: keyboards/sharkoon/skiller_sgk50_s2/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    [0] = LAYOUT(
+        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PAUS, KC_HOME, KC_END,    KC_PGUP,  KC_PGDN,  KC_DEL,
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_NUM,    KC_PSLS,  KC_PAST,  KC_PMNS,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_P7,     KC_P8,    KC_P9,    KC_PPLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           KC_P4,     KC_P5,    KC_P6,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT, KC_UP,   KC_P1,     KC_P2,    KC_P3,    KC_PENT,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT,   KC_P0,    KC_PDOT
+    ),
+
+    [1] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s2/keymaps/via/keymap.c:17]:
```suggestion
    [0] = LAYOUT_all(
        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PAUS, KC_HOME, KC_END,    KC_PGUP,  KC_PGDN,  KC_DEL,
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_NUM,    KC_PSLS,  KC_PAST,  KC_PMNS,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_P7,     KC_P8,    KC_P9,    KC_PPLS,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           KC_P4,     KC_P5,    KC_P6,
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT, KC_UP,   KC_P1,     KC_P2,    KC_P3,    KC_PENT,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT,   KC_P0,    KC_PDOT
    ),

    [1] = LAYOUT_all(
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
Conforms to PR Checklist regarding a PCB with multiple layouts

**Comment by itarze** [comment]
> > > Does this PCB have ISO (form factor) support?
> 
> > > in particular - ISO Hash and ISO Enter keys
> 
> > 
> 
> > yes.
> 
> 
> 
> Great stuff.
> 
> Would I be correct that the matrix co-ordinates for the following keys are identical?:
> 
> - ANSI Backslash <> ISO Hash
> 
> - ANSI Enter <> ISO Enter

bingo.

File: keyboards/sharkoon/skiller_sgk50_s2/info.json [hunk 7]
```diff
@@ -0,0 +1,411 @@
...
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x3662",
+        "suspend_wakeup_delay": 1000,
+        "vid": "0x6332"
+    },
+    "ws2812": {
+        "pin": "A8"
+    },
+    "community_layouts": ["96_ansi"],
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s2/info.json:199]:
```suggestion
    "community_layouts": ["96_ansi", "96_iso"],
```

File: keyboards/sharkoon/skiller_sgk50_s2/info.json [hunk 8]
```diff
@@ -0,0 +1,411 @@
...
+                {"matrix": [5, 10], "x": 10, "y": 5},
+                {"matrix": [5, 11], "x": 11, "y": 5},
+                {"matrix": [5, 12], "x": 12, "y": 5},
+                {"matrix": [5, 13], "x": 13, "y": 5},
+                {"matrix": [5, 14], "x": 14, "y": 5},
+                {"matrix": [5, 15], "x": 15, "y": 5},
+                {"matrix": [5, 16], "x": 16, "y": 5},
+                {"matrix": [5, 17], "x": 17, "y": 5}
+            ]
+        }
```
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s2/info.json:409]:
```suggestion
        },
        "LAYOUT_96_iso": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0},
                {"matrix": [0, 14], "x": 14, "y": 0},
                {"matrix": [0, 15], "x": 15, "y": 0},
                {"matrix": [0, 16], "x": 16, "y": 0},
                {"matrix": [0, 17], "x": 17, "y": 0},
                {"matrix": [0, 18], "x": 18, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1},
                {"matrix": [1, 1], "x": 1, "y": 1},
                {"matrix": [1, 2], "x": 2, "y": 1},
                {"matrix": [1, 3], "x": 3, "y": 1},
                {"matrix": [1, 4], "x": 4, "y": 1},
                {"matrix": [1, 5], "x": 5, "y": 1},
                {"matrix": [1, 6], "x": 6, "y": 1},
                {"matrix": [1, 7], "x": 7, "y": 1},
                {"matrix": [1, 8], "x": 8, "y": 1},
                {"matrix": [1, 9], "x": 9, "y": 1},
                {"matrix": [1, 10], "x": 10, "y": 1},
                {"matrix": [1, 11], "x": 11, "y": 1},
                {"matrix": [1, 12], "x": 12, "y": 1},
                {"matrix": [1, 13], "x": 13, "y": 1, "w": 2},
                {"matrix": [1, 15], "x": 15, "y": 1},
                {"matrix": [1, 16], "x": 16, "y": 1},
                {"matrix": [1, 17], "x": 17, "y": 1},
                {"matrix": [1, 18], "x": 18, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.5},
                {"matrix": [2, 1], "x": 1.5, "y": 2},
                {"matrix": [2, 2], "x": 2.5, "y": 2},
                {"matrix": [2, 3], "x": 3.5, "y": 2},
                {"matrix": [2, 4], "x": 4.5, "y": 2},
                {"matrix": [2, 5], "x": 5.5, "y": 2},
                {"matrix": [2, 6], "x": 6.5, "y": 2},
                {"matrix": [2, 7], "x": 7.5, "y": 2},
                {"matrix": [2, 8], "x": 8.5, "y": 2},
                {"matrix": [2, 9], "x": 9.5, "y": 2},
                {"matrix": [2, 10], "x": 10.5, "y": 2},
                {"matrix": [2, 11], "x": 11.5, "y": 2},
                {"matrix": [2, 12], "x": 12.5, "y": 2},
                {"matrix": [2, 15], "x": 15, "y": 2},
                {"matrix": [2, 16], "x": 16, "y": 2},
                {"matrix": [2, 17], "x": 17, "y": 2},
                {"matrix": [2, 18], "x": 18, "y": 2, "h": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 1.75},
                {"matrix": [3, 2], "x": 1.75, "y": 3},
                {"matrix": [3, 3], "x": 2.75, "y": 3},
                {"matrix": [3, 4], "x": 3.75, "y": 3},
                {"matrix": [3, 5], "x": 4.75, "y": 3},
                {"matrix": [3, 6], "x": 5.75, "y": 3},
                {"matrix": [3, 7], "x": 6.75, "y": 3},
                {"matrix": [3, 8], "x": 7.75, "y": 3},
                {"matrix": [3, 9], "x": 8.75, "y": 3},
                {"matrix": [3, 10], "x": 9.75, "y": 3},
                {"matrix": [3, 11], "x": 10.75, "y": 3},
                {"matrix": [3, 12], "x": 11.75, "y": 3},
                {"matrix": [2, 13], "x": 12.75, "y": 3},
                {"matrix": [3, 13], "x": 13.75, "y": 2, "w": 1.25, "h": 2},
                {"matrix": [3, 15], "x": 15, "y": 3},
                {"matrix": [3, 16], "x": 16, "y": 3},
                {"matrix": [3, 17], "x": 17, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [4, 1], "x": 1.25, "y": 4},
                {"matrix": [4, 2], "x": 2.25, "y": 4},
                {"matrix": [4, 3], "x": 3.25, "y": 4},
                {"matrix": [4, 4], "x": 4.25, "y": 4},
                {"matrix": [4, 5], "x": 5.25, "y": 4},
                {"matrix": [4, 6], "x": 6.25, "y": 4},
                {"matrix": [4, 7], "x": 7.25, "y": 4},
                {"matrix": [4, 8], "x": 8.25, "y": 4},
                {"matrix": [4, 9], "x": 9.25, "y": 4},
                {"matrix": [4, 10], "x": 10.25, "y": 4},
                {"matrix": [4, 11], "x": 11.25, "y": 4},
                {"matrix": [4, 12], "x": 12.25, "y": 4, "w": 1.75},
                {"matrix": [4, 14], "x": 14, "y": 4},
                {"matrix": [4, 15], "x": 15, "y": 4},
                {"matrix": [4, 16], "x": 16, "y": 4},
                {"matrix": [4, 17], "x": 17, "y": 4},
                {"matrix": [4, 18], "x": 18, "y": 4, "h": 2},
                {"matrix": [5, 0], "x": 0, "y": 5, "w": 1.25},
                {"matrix": [5, 1], "x": 1.25, "y": 5, "w": 1.25},
                {"matrix": [5, 2], "x": 2.5, "y": 5, "w": 1.25},
                {"matrix": [5, 6], "x": 3.75, "y": 5, "w": 6.25},
                {"matrix": [5, 10], "x": 10, "y": 5},
                {"matrix": [5, 11], "x": 11, "y": 5},
                {"matrix": [5, 12], "x": 12, "y": 5},
                {"matrix": [5, 13], "x": 13, "y": 5},
                {"matrix": [5, 14], "x": 14, "y": 5},
                {"matrix": [5, 15], "x": 15, "y": 5},
                {"matrix": [5, 16], "x": 16, "y": 5},
                {"matrix": [5, 17], "x": 17, "y": 5}
            ]
        }
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
add Community Layout support: 96_iso

**Review by dunk2k — CHANGES_REQUESTED** [review]
add EoF

File: keyboards/sharkoon/skiller_sgk50_s2/readme.md
Comment (dunk2k) [inline keyboards/sharkoon/skiller_sgk50_s2/readme.md:1]: 1. Is an image of the PCB/keyboard in question available to insert in this file?
2. is there availability information? i.e. site(s) or contact details of where this keyboard can be purchased

File: keyboards/sharkoon/skiller_sgk50_s2/readme.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
+# SHARKOON SKILLER SGK50 S2
+
```
Comment (itarze) [inline keyboards/sharkoon/skiller_sgk50_s2/readme.md:2]:
```suggestion

![ISO](https://i.imgur.com/itgy5Bk.png)

```

File: keyboards/sharkoon/skiller_sgk50_s2/readme.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+![ISO](https://i.imgur.com/itgy5Bk.png)
+
+* Keyboard Maintainer: [JoyLee](https://github.com/itarze)
+* Hardware Supported: SHARKOON SKILLER SGK50 S2 PCB
```
Comment (itarze) [inline keyboards/sharkoon/skiller_sgk50_s2/readme.md:6]:
```suggestion
* Hardware Supported: [SHARKOON SKILLER SGK50 S2 PCB](www.sharkoon.com)
```

File: keyboards/sharkoon/skiller_sgk50_s2/readme.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
+# SHARKOON SKILLER SGK50 S2
+
+![ISO](https://i.imgur.com/itgy5Bk.png)
+
+* Keyboard Maintainer: [JoyLee](https://github.com/itarze)
+* Hardware Supported: [SHARKOON SKILLER SGK50 S2 PCB](www.sharkoon.com)
```
Comment (itarze) [inline keyboards/sharkoon/skiller_sgk50_s2/readme.md:6]:
```suggestion
* Hardware Supported: [SHARKOON SKILLER SGK50 S2 PCB](https://zh-hant.sharkoon.com/)
```

**Review by drashna — APPROVED** [review]

File: keyboards/sharkoon/skiller_sgk50_s2/info.json [hunk 12]
```diff
@@ -0,0 +1,516 @@
...
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "rgblight": false
```
Comment (waffle87) [inline keyboards/sharkoon/skiller_sgk50_s2/info.json:22]:
```suggestion
        "rgb_matrix": true
```

**Review by dunk2k — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**